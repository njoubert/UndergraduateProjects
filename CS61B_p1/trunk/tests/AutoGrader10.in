Jura is not the dummy.
Wolodymyr is not the dummy.
Wolodymyr is not the lawyer.
Jura is not the diplomat.
Dummy lives around here.
Who is the dummy?
What does Dummy do?
What do you know about Jura?
What do you know about Wolodymyr?

