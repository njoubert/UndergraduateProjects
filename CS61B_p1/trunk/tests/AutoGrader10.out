1. Jura is not the dummy.
2. Wolodymyr is not the dummy.
3. Wolodymyr is not the lawyer.
4. Jura is not the diplomat.
5. Dummy lives around here.
Q: Who is the dummy?
A: Dummy is the dummy.
Q: What does Dummy do?
A: Dummy is the dummy.
Q: What do you know about Jura?
A: Jura is the lawyer.
Q: What do you know about Wolodymyr?
A: Wolodymyr is the diplomat.
